<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>stud_mas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QPushButton" name="delete_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>70</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="edit_lineEdit">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>520</y>
          <width>351</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditSelect">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>40</y>
          <width>251</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QListWidget" name="names_list">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>100</y>
          <width>256</width>
          <height>451</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="edit_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>70</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Select_button">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>70</y>
          <width>251</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>520</y>
          <width>351</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="Send_button">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>520</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
       <widget class="QPushButton" name="resend_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>70</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Resend</string>
        </property>
       </widget>
       <widget class="QListWidget" name="chatSpace">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>100</y>
          <width>451</width>
          <height>411</height>
         </rect>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="autoScroll">
         <bool>true</bool>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="send_edit_button">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>520</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
       <widget class="QPushButton" name="send_image">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>70</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Send Img</string>
        </property>
       </widget>
       <widget class="QLabel" name="UserNameLabel">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>40</y>
          <width>401</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="graffiti_button">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>30</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Send GRRFT</string>
        </property>
       </widget>
       <zorder>send_edit_button</zorder>
       <zorder>delete_button</zorder>
       <zorder>edit_lineEdit</zorder>
       <zorder>lineEditSelect</zorder>
       <zorder>names_list</zorder>
       <zorder>edit_button</zorder>
       <zorder>Select_button</zorder>
       <zorder>lineEdit</zorder>
       <zorder>Send_button</zorder>
       <zorder>resend_button</zorder>
       <zorder>chatSpace</zorder>
       <zorder>send_image</zorder>
       <zorder>UserNameLabel</zorder>
       <zorder>graffiti_button</zorder>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QLineEdit" name="lineEditUserName">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>160</y>
          <width>281</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string comment="Enter your UserName"/>
        </property>
       </widget>
       <widget class="QPushButton" name="Connect_button">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>200</y>
          <width>281</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="set_red_button">
            <property name="text">
             <string>Red</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="set_green_button">
            <property name="text">
             <string>Green</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="set_blue_button">
            <property name="text">
             <string>Blue</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QRadioButton" name="stream_flag">
            <property name="text">
             <string>Stream</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Graffiti_space" name="widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="send_graffiti_button">
            <property name="text">
             <string>Send</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Graffiti_space</class>
   <extends>QWidget</extends>
   <header>Graffiti_space.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
